### ISSUE TITLE
User has preffered Ledger

### ISSUE THREAD

**Issue by matthiasbeyer** [cashctrl/cashctrl#16]

**Comment by TheNeikos** [comment]
Is the use case that it is first for auto completion?

**Comment by matthiasbeyer** [comment]
Yes. If the user does "new transaction" on the dashboard, the application automatically selects the preferred ledger for this and the user can change this, of course...
